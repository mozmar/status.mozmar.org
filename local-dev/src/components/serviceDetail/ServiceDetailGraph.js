import React, { Component } from 'react';

class ServiceDetailGraph extends Component {
    static propTypes = {
        url: React.PropTypes.string.isRequired
    }

    render() {
        return (
            <div className="service-detail-alert-graph-wrapper">
                <iframe className="service-detail-alert-graph" src={this.props.url} />
            </div>
        );
    }
}

export default ServiceDetailGraph;
