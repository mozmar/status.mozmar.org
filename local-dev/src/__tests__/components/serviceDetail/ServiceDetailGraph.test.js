import React from 'react';
import renderer from 'react-test-renderer';

import ServiceDetailGraph from '../../../components/serviceDetail/ServiceDetailGraph';

test('render ServiceDetailGraph', () => {
    const component = renderer.create(
        <ServiceDetailGraph url="http://www.mozilla.org/" />
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
