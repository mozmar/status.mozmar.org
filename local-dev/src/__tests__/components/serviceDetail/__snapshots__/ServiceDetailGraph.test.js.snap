exports[`test render ServiceDetailGraph 1`] = `
<div
  className="service-detail-alert-graph-wrapper">
  <iframe
    className="service-detail-alert-graph"
    src="http://www.mozilla.org/" />
</div>
`;
